dubbo-rpc/dubbo-rpc-rest/src/main/java/org/apache/dubbo/rpc/protocol/rest/RestProtocol.java

```diff
@@ -113,6 +122,15 @@ public void afterUnExport() {
         return exporter;
     }
 
+    private void checkJsonCompatibility(Class<?> clazz) throws RpcException {
+        boolean compatibility = JsonCompatibilityUtil.checkClassCompatibility(clazz);
+        if (!compatibility) {
+            List<String> unsupportedMethods = JsonCompatibilityUtil.getUnsupportedMethods(clazz);
+            logger.error("Interface {} does not support json serialization, the specific methods are {}", clazz, unsupportedMethods);
+            throw new RpcException(String.format("Interface %s does not support json serialization", clazz));
```

Comment:
Print logs or affect startup, should be decided by the user, it should be a configuration, configuration json check levels can be ignore (no check), log, startup error, respectively can correspond to close, alarm, strict

Reply:
done!
In the new version, user can specify the implementation mode by specifying attribute `json-check-level`.  The optional values are `ignore`, `log`, or `startup-error`. 
If no value is specified, the default implementation mode is `startup-error`.